tests/haskell/GhcModBearTest.py

```diff
@@ -0,0 +1,96 @@
+from queue import Queue
+
+from bears.haskell.GhcModBear import GhcModBear
+from coalib.testing.LocalBearTestHelper import LocalBearTestHelper
+from coalib.testing.LocalBearTestHelper import verify_local_bear
+from coalib.testing.BearTestHelper import generate_skip_decorator
+from coalib.settings.Section import Section
+from coalib.settings.Setting import Setting
+
+
+# A simple hello world program!
+hello_world_file = """
+main :: IO()
+main = putStrLn "Hello World!"
+"""
```

Comment:
for example you could change these to `""".splitLines()` to implement what makamn2 advised